Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -467,14 +467,10 @@ def _define_symbols():
 
 
 class OWScatterPlotGraph(gui.OWComponent, ScaleScatterPlotData):
-    attr_color = ContextSetting(
-        None, ContextSetting.OPTIONAL, exclude_metas=False)
-    attr_label = ContextSetting(
-        None, ContextSetting.OPTIONAL, exclude_metas=False)
-    attr_shape = ContextSetting(
-        None, ContextSetting.OPTIONAL, exclude_metas=False)
-    attr_size = ContextSetting(
-        None, ContextSetting.OPTIONAL, exclude_metas=False)
+    attr_color = ContextSetting(None, required=ContextSetting.OPTIONAL)
```

Comment:
ContextSetting.OPTIONAL was passed to the not_attibute parameter, although the constant is meant for the required parameter.